homestead.md

```diff
@@ -45,8 +45,6 @@ Laravel Homestead is an official, pre-packaged Vagrant box that provides you a w
 
 Homestead runs on any Windows, Mac, or Linux system, and includes Nginx, PHP, MySQL, PostgreSQL, Redis, Memcached, Node, and all of the other goodies you need to develop amazing Laravel applications.
 
-> {note} If you are using Windows, you may need to enable hardware virtualization (VT-x). It can usually be enabled via your BIOS. If you are using Hyper-V on a UEFI system you may additionally need to disable Hyper-V in order to access VT-x.
```

Comment:
We shouldn't remove this, it's always happening to someone.

Reply:
I don't agree. It is happening to a few people that want to use very old version of solutions like Virtualbox 5.x or older.

Disabling Hyper-V, you disable support for Android Emulator and Docker for Windows. Both on current versions are working with Hyper-V.

We should support the majority, not the minority. I think those that use old versions should know what they do.